pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -346,6 +346,23 @@ func skipIfHTTPSNotSupported(t *testing.T, _ testContext) {
 	}
 }
 
+func testTLSClassification(t *testing.T, tr *Tracer, clientHost, targetHost, serverHost string) {
+	t.Run("TLS", func(t *testing.T) {
```

Comment:
I think the wrapper here is redundant, as we have a wrapper per sub-test

Reply:
The reason for this "TLS" subtest is to avoid naming conflicts and confusion when looking at the test output:
- currently we already have this test: `TestUSMSuite/.*/TestProtocolClassification/.*/amqp`
- With this subtest, we now have `TestUSMSuite/.*/TestProtocolClassification/.*/TLS/AMQP`
- Without it, we would have `TestUSMSuite/.*/TestProtocolClassification/.*/AMQP` (or, if I had used lowercase for amqp, it would probably show up like this `TestUSMSuite/.*/TestProtocolClassification/.*/amqp#01`).

I think this subtest makes it clear when reading the test report that we are running the tests for AMQP on top of TLS.

Reply:
nit: let's use a single pattern - either AMQP or amqp